ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/influxdb.schema.yml

```
@@ -0,0 +1,79 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  influxdb:
+    type: dict
+    keys:
+      vrf:
+        type: str
+      destinations:
+        type: list
+        primary_key: name
+        description: Configure telemetry output destinations.
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: InfluxDB connection name.
+            database:
+              type: str
+            policy:
+              type: str
+              description: Confiure data retention policy.
+            url:
+              type: str
+              description: Configure URL for the destination.
+            username:
+              type: str
+              description: Configure login details with destination.
+            password:
+              type: str
+              description: Assign login password
+            encryption:
+              type: str
+              convert_types:
+                - int
+              description: |-
+                0: Indicates that keystring is not encrypted.
+                7: Speciefies that HIDDEN key will follow.
+                8a: Speciefies that AES-256-GCM encrypted key will follow.
```

Adjusted the name to be consistent with other AVD models.
Also we normally use type 7 by default.
```suggestion
            password_type:
              type: str
              convert_types:
                - int
              valid_values: ["0", "7", "8a"]
              default: "7"
```

---

Fixed.